concent_api/gatekeeper/tests/test_utils.py

```diff
@@ -0,0 +1,31 @@
+import json
+
+from django.http                    import JsonResponse
+from django.test                    import TestCase
+
+from gatekeeper.utils               import gatekeeper_access_denied_response
+
+class GatekeeperAccessDeniedResponseTest(TestCase):
+
+    def setUp(self):
+        self.message     = "Missing 'Authorization' header."
+        self.path        = 'blender/benchmark/test_task/scene-Helicopter-27-cycles.blend'
+        self.client_key  = '85cZzVjahnRpUBwm0zlNnqTdYom1LF1P1WNShLg17cmhN2UssnPrCjHKTi5susO3wrr/q07eswumbL82b4HgOw=='
+
+    def test_response_should_return_appropriate_body_and_header(self):
```

Comment:
You're testing a function called `gatekeeper_access_denied_response` so the test should be called `test_gatekeeper_access_denied_response_should_return_appropriate_body_and_headers()` to follow our convention.